packages/connector-sendgrid-mail/src/mock.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  Content,
+  ContextType,
+  EmailData,
+  Personalization,
+  PublicParameters,
+  SendGridMailConfig,
+} from './types';
+
+const toEmailData: EmailData[] = [{ email: 'foo@logto.io' }];
+const fromEmailData: EmailData = { email: 'noreply@logto.test.io', name: 'Logto Test' };
```

Comment:
maybe better name them `sender` and `receiver`

Reply:
fixed.